code-signer/action.yml

```diff
@@ -85,18 +88,20 @@ runs:
             -totp_secret="${{ inputs.totp_secret }}" \
             -output_dir_path="$OUTPUT_DIR" \
             -input_file_path="$INSTALLER_FILE"
+          cp ${INSTALLER_FILE} ${INSTALLER_FILE}.unsigned # keep unsigned for backup
```

Comment:
Can we move that before the code signing command? just to be sure we're copying an unsigned artifact